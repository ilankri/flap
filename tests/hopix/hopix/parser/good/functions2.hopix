fun f (x : int, y : int) : int = x + y
