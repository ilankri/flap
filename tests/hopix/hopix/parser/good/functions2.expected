
fun f (x : int, y : int) = (`+ (x, y) : int)
